Odd number or arguments passed as key-value pairs for logging on log.Error

If you try to start this process with something that fails and makes a call to `log.Error`  (like, trying to start two mbop processes) , something that is hinted as an incorrect logger usage is displayed: 

```
$ ./mbop 
2022-05-31T20:26:09.194+0200    DPANIC  mbop/main.go:491        odd number of arguments passed as key-value pairs for logging   {"ignored key": "server couldn't start"}
main.main
        /home/vmugicag/dev/projects/mbop/main.go:491
runtime.main
        /usr/lib/golang/src/runtime/proc.go:250
panic: odd number of arguments passed as key-value pairs for logging

goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0001240c0, {0xc0000ec440, 0x1, 0x1})
        /home/vmugicag/go/pkg/mod/go.uber.org/zap@v1.21.0/zapcore/entry.go:232 +0x44c
go.uber.org/zap.(*Logger).DPanic(0x706265?, {0x716c46?, 0x6a9920?}, {0xc0000ec440, 0x1, 0x1})
        /home/vmugicag/go/pkg/mod/go.uber.org/zap@v1.21.0/logger.go:220 +0x59
github.com/go-logr/zapr.(*zapLogger).handleFields(0xc0000a6db0, 0xffffffffffffffff, {0xc0000a0e40, 0x1, 0xc0000d5e68?}, {0xc0000ec3c0?, 0x1, 0x0?})
        /home/vmugicag/go/pkg/mod/github.com/go-logr/zapr@v1.2.3/zapr.go:147 +0xd31
github.com/go-logr/zapr.(*zapLogger).Error(0xc0000a6db0, {0x786dc0?, 0xc0000d8190}, {0x704aad?, 0xc0000d5ef8?}, {0xc0000a0e40, 0x1, 0x1})
        /home/vmugicag/go/pkg/mod/github.com/go-logr/zapr@v1.2.3/zapr.go:216 +0x1ac
github.com/go-logr/logr.Logger.Error({{0x789cd0?, 0xc0000a6db0?}, 0xc0000d5f70?}, {0x786dc0, 0xc0000d8190}, {0x704aad, 0x6}, {0xc0000a0e40, 0x1, 0x1})
        /home/vmugicag/go/pkg/mod/github.com/go-logr/logr@v1.2.2/logr.go:270 +0xba
main.main()
        /home/vmugicag/dev/projects/mbop/main.go:491 +0xef
```

--- a/cmd/mbop/mbop.go
+++ b/cmd/mbop/mbop.go
@@ -35,6 +35,6 @@ func main() {
 
 	l.Log.Info("Starting MBOP Server on :8090")
 	if err := srv.ListenAndServe(); err != nil {
-		l.Log.Error(err, "reason", "server couldn't start")
+		l.Log.Error(err, "server couldn't start")
 	}
 }

